src/test/java/uk/gov/hmcts/ccd/domain/service/accessprofile/filter/FilterRoleAssignmentsImplTest.java

```diff
@@ -0,0 +1,170 @@
+package uk.gov.hmcts.ccd.domain.service.accessprofile.filter;
+
```

Comment:
we need more testing for the filtering stage. It's something well defined where I don't expect changes, so we can add tests now rather than later on